images/secrets-store-csi-driver-provider-gcp/configs/latest.apko.yaml

```diff
@@ -15,7 +15,7 @@ accounts:
   users:
     - username: secrets-store-csi-driver-provider-gcp
       uid: 65532
```

Comment:
Should we even bother having a nonroot user here, if it's not going to be practically usable? (Ditto for the other image)

Reply:
can resolve in follow up